test.py

```diff
@@ -0,0 +1,94 @@
+#fasta
+fasta = open("S_cerevisiae/saccharomyces_cerevisiae_R64-1-1_20110208_sequences.fasta", "r")
+
+seq = fasta.read()
+seqlist = seq.split("\n")
+seqlist_new = []
+name = []
+count = -1
+
+for i in range(len(seqlist)):
+    if ">" in seqlist[i]:
+        name.append(seqlist[i][1:])
+        count += 1
+        seqlist_new.append("")
+    else:
+        seqlist_new[count] += seqlist[i]
+
+
+#gff
+
+import re
+
+gff = open("S_cerevisiae/saccharomyces_cerevisiae_R64-1-1_20110208_annotation.gff", "r")
+f = gff.readlines()
+
+# #で始まる行を削除
+for i in range(len(f)):
+    if f[0][0] == "#":
+        f.pop(0)
+    else:
+        break
+
+#スペースで分割
+inf = []
+for i in range(len(f)):
+    inf.append("")
+    inf[i] = f[i].split("\t")
+
+
+gene = []
+cds = []
+cds_count = 0
+for i in range(len(f)):  
+    
+    if inf[i][2] == "CDS":
+        cds.append("")
+        cds_name = re.findall(r"Parent=(.*?);", inf[i][8])
```

Comment:
```suggestion
        cds_name = re.findall(r"Parent=(.+?);", inf[i][8])
```
ParentのIDが何もないことはないので、1文字以上マッチングの`+`を使うといい